src/storm/mesos/MesosCommon.java

```diff
@@ -39,6 +40,22 @@
   public static final String SUPERVISOR_ID = "supervisorid";
   public static final String ASSIGNMENT_ID = "assignmentid";
 
+  
+  public static String ipAddressFromAssignmentId(String assignmentId) {
+        int last = assignmentId.lastIndexOf("-");
+        String ip = assignmentId.substring(last + 1);
+        LOG.info("IPAddress: " + ip);
```

Comment:
Please give me an example (fake is fine) value that you are parsing.  That parsing would break my system, I think we only have hostnames in the assignmentId, no IPs.